### ISSUE TITLE
Range type issue when ZDDX defines min > max values for configuration parameter

### ISSUE THREAD

**Issue by kozlowski-mathias** [Z-Wave-Me/ExpertUI#61]
ZDDX file for Fibaro Motion Sensor (540-...) defines min/max values for parameter 66 as from="8001" to="7fff". This is correct, Fibaro device offset temperature with this signed value. Your code in function configCont() of controllers.js file does not handle this...

**Comment by kozlowski-mathias** [comment]
Can't reproduce the problem I have. No idea what changed.

**Comment by kozlowski-mathias** [comment]
For info, can't achieve a "Save into Device" action with my Fibaro Motion Sensor with releases 2.0.1-rc8 and 2.0.1-rc9. I should downgrade to -rc7.

**Comment by kozlowski-mathias** [comment]
OK, my Fibaro Motion Sensor parameter 66 value is 32769, input box info tip shows (from ZDDX) min: 32769 and max: 32767 but the response from POST HTTP request ...commandClasses[112].data[66] shows the correct 0 current value. There must be a = max(v, min) somewhere in display code and this is not compatible with Fibaro parameter range choice.

**Comment by PoltoS** [comment]
Indeed, we broke some stuff in queueing. Please install latest rc9-3-.... This will fix the issue

**Comment by kozlowski-mathias** [comment]
OK, I wasn't sure of the stability of -3-* release, bad bet...

**Comment by PoltoS** [comment]
Is it still an issue?

**Comment by dolpheen** [comment]
Yes, It's stiil an issue.
In expertCommandInput directive (in directives.js) negative values for Fibaro sensor handled as poditives ones, also there is a check 'var setVal = (value ? value : min);' that sets the min if value is 0 (that's why 32769 is shown in interface)
